benchmark/clblast/main.cpp

```diff
@@ -1,5 +1,5 @@
 #include "utils.hpp"
-#include "utils/extract_vendor_type.hpp"
+#include <common/extract_vendor_type.hpp>"
```

Comment:
```suggestion
#include "common/extract_vendor_type.hpp"
```

Looks like there's a typo there, but also it seems to me like double quotes would be preferable for including this header, also elsewhere.

Reply:
So, I transformed `common` into a library with it's own target. The quotes would mean this is a local file no?

Reply:
See d786661f8318649ab60b2b32fe897fdad333edb7 for the fix but I say we stick with `<>` for includes outside of target.

Reply:
Makes sense to me, if it's meant to be a utility